akka-docs/release-train-issue-template.md

```diff
@@ -31,7 +31,7 @@ Key links:
 ### Check availability
 
 - [ ] Check [API](https://doc.akka.io/api/akka/$VERSION$/) documentation
```

Comment:
what abaout api and japi?

Reply:
I believe they can stay as is.

Reply:
Ah, I now think you mean akka-core for api and japi. Right. That should change.